staging/src/k8s.io/api/resource/v1beta1/types.go

```diff
@@ -725,7 +725,7 @@ type ResourceClaimStatus struct {
 
 // ReservedForMaxSize is the maximum number of entries in
 // claim.status.reservedFor.
-const ResourceClaimReservedForMaxSize = 32
+const ResourceClaimReservedForMaxSize = 256
```

Comment:
nit: fix the go doc while touching this.

Reply:
same for internal types

Reply:
Which go doc is outdated? I changed 32 to 256 in the field docs in all three files.